examples/threathunter/threat_intelligence/config.yml

```diff
@@ -0,0 +1,70 @@
+sites:
+    alienvault:
+        # the feed_id of the ThreatHunter Feed you want to send ThreatIntel to
+        feed_id: 5thTpazzRdWhQeHnjWUfbA
+
+        # the address of the site (only server ip or dns; don't put https:// or a trailing slash)
+        site: otx.alienvault.com
+
+        # the path of the site for discovering what services are available
+        # this is supplied by your taxii provider
+        discovery_path: /taxii/discovery
+
+        # the path of the site for listing what collections are available to you
+        # this is supplied by your taxii provider
+        collection_management_path: /taxii/collections
+
+        # the path of the site for polling a collection
+        # this is supplied by your taxii provider
+        poll_path: /taxii/poll
+
+        # change to true if you require https for your TAXII service connection
+        use_https: True
+
+        # by default, we validate SSL certificates. Turn this off by setting ssl_verify: False
+        ssl_verify: False
+
+        # (optional) if you need SSL certificates for authentication, set the path of the
+        # certificate and key here. Please leave blank to ignore.
+        cert_file:
+        key_file:
+
+        # how to score each result. Accepts values [1,10], and defaults to 5
+        default_score: 5
+
+        # (optional) username for authorization with your taxii provider
+        username: 0dd92ced56580311a769a496cd1fa72533f7b5d1f4c1b1b6aa7a35425baa97f6
+
+        # (optional) password for authorization with your taxii provider
+        password:
+
+        # (optional) specify which collections to convert to feeds (comma-delimited)
+        collections: 'user_AlienVault'
+
+        # (optional) the start date for which to start requesting data.
+        # Defaults to 2019-01-01 00:00:00 if you supply nothing
+        start_date: 2019-11-01 00:00:00
```

Comment:
@llyon-cb Can we also make this an input argument in case we wanted to automate the script

Reply:
I will add the option for command line arguments to allow greater automation

Reply:
The user now has the option to supply command line arguments to update `start_date` programatically